frontend/app/pods/admin/teams/template.hbs

```diff
@@ -7,6 +7,7 @@
 <div class="row">
   <div class="small-5 columns">
     {{#each model as |team|}}
+      {{#if team.name}}
```

Comment:
from my understanding - the Amber.store keeps a copy if each new opened model before it calls the submit-action and does the save() function.
I'm not sure how to prevent this, as I'm fairly new to the concept of ember.
I'll try to read up on it, but it'll take some time. I'll add this as a fronend bug.